server/modules/instantiation/jobs.js

```diff
@@ -457,6 +460,45 @@ module.exports = function (config, portProvider, interpolationHelper, buildInsta
         }
     }
 
+    class RunAfterBuildTaskJobExecutor extends JobExecutor {
+        supports(job) {
+            return (job instanceof RunAfterBuildTaskJob);
+        }
+
+        execute(job) {
+            return new Promise((resolve, reject) => {
+                var { componentInstance, scriptPath } = job;
+                var componentId = componentInstance.id;
+                var scriptFullPath = path.join(componentInstance.fullPath, scriptPath);
+                var env = _.extend(
+                    {},
+                    componentInstance.buildInstance.environmentalVariables,
+                    {
+                        FEAT__INSTANCE_ID: componentInstance.buildInstance.id,
+                    }
+                )
+
+                componentInstance.log(`Running ${scriptPath}.`);
+
+                exec(scriptFullPath, {
+                    cwd: componentInstance.fullPath,
+                    env: env,
+                }, (error, stdout) => {
+                    if (error) {
+                        componentInstance.error(`Failed to execute ${scriptPath}`);
+                        reject(error);
```

Comment:
here I just add simple messsage that the script has failed and reject, nothing will be wiped out as I don't use stderr (not to duplicate error messages)